src/D2L.CodeStyle.Analyzers/Async/Generator/FileCollector.cs

```diff
@@ -82,6 +82,9 @@ internal sealed class FileCollector {
 		public string CollectSource() {
 			// TODO: Remove this and modify XML param elements in generator when changed/removed
 			m_out.AppendLine( "#pragma warning disable CS1572" );
+			// This allows us to copy+paste annotations but otherwise does not emit diagnostics
+			// otherwise we get "CS8669: The annotation for nullable reference types should only be used in code within a '#nullable' annotations context. Auto-generated code requires an explicit '#nullable' directive in source."
+			m_out.AppendLine( "#nullable annotations" );

```

Comment:
If a project specifies its own nullable context explicitly at the project level, how does this interact?

Reply:
This only applies to this file. It seems that we need to expliclitly set it per-file for generated code regardless, otherwise its equivalent to the status quo.